markdown/assignments/sheet06/index.md

```diff
@@ -2,25 +2,108 @@
 type: assignment
 title: "Blatt 06: Perzeptron, Lineare Regression"
 author: "Canan Yıldız (Türkisch-Deutsche Universität)"
+points: 10
 hidden: true
 weight: 6
-sketch: true
+sketch: false
 ---
 
+<style type="text/css">
```

Comment:
funktioniert das zusammen mit hugo? 

sollten wir das in die gemeinsamen style-definitionen aufnehmen?

Reply:
das hat bei mir lokal funktioniert mit hugo.

in die style-definitionen aufzunehmen wäre natürlich eine elegantere lösung.
aber so, dass es nicht alle ul-elemente beeinflusst, oder?

Reply:
guter einwand. vermutlich müsste man dann um den kompletten aufgabentext ein extra div machen, damit man im css danach filtern kann und diese änderungen nur bei aufgaben wirksam werden. das klingt erstmal nach einer größeren operation, ich mache uns mal ein ticket auf für später und du löst das in der zwischenzeit erstmal so lokal?

Reply:
https://github.com/cagix/Hugo-Lecture-Layouts/issues/36